dockerfiles/config/config-docker.ttl

```diff
@@ -0,0 +1,146 @@
+    # customize the service name
+    skosmos:serviceName "Skosmos" ;
+    # customize the base element. Set this if the automatic base url detection doesn't work. For example setups behind a proxy.
+    # skosmos:baseHref "http://localhost/Skosmos/" ;
+    # interface languages available, and the corresponding system locales
+    skosmos:languages (
+        [ rdfs:label "ar" ; rdf:value "ar_AE.utf8" ]
+        [ rdfs:label "da" ; rdf:value "da_DK.utf8" ]
+        [ rdfs:label "de" ; rdf:value "de_DE.utf8" ]
+        [ rdfs:label "en" ; rdf:value "en_GB.utf8" ]
+        [ rdfs:label "en_US" ; rdf:value "en_US.utf8" ]
+        [ rdfs:label "es" ; rdf:value "es_ES.utf8" ]
+        [ rdfs:label "fa" ; rdf:value "fa_IR.utf8" ]
+        [ rdfs:label "fi" ; rdf:value "fi_FI.utf8" ]
+        [ rdfs:label "fr" ; rdf:value "fr_FR.utf8" ]
+        [ rdfs:label "it" ; rdf:value "it_IT.utf8" ]
+        [ rdfs:label "nb" ; rdf:value "nb_NO.utf8" ]
+        [ rdfs:label "nl" ; rdf:value "nl_NL.utf8" ]
+        [ rdfs:label "nn" ; rdf:value "nn_NO.utf8" ]
+        [ rdfs:label "pl" ; rdf:value "pl_PL.utf8" ]
+        [ rdfs:label "pt" ; rdf:value "pt_PT.utf8" ]
+        [ rdfs:label "pt_BR" ; rdf:value "pt_BR.utf8" ]
+        [ rdfs:label "ru" ; rdf:value "ru_RU.utf8" ]
+        [ rdfs:label "sv" ; rdf:value "sv_SE.utf8" ]
+        [ rdfs:label "zh" ; rdf:value "zh_CN.utf8" ]
+    ) ;
+    # how many results (maximum) to load at a time on the search results page
+    skosmos:searchResultsSize 20 ;
+    # how many items (maximum) to retrieve in transitive property queries
+    skosmos:transitiveLimit 1000 ;
+    # whether or not to log caught exceptions
+    skosmos:logCaughtExceptions false ;
+    # set to TRUE to enable logging into browser console
+    skosmos:logBrowserConsole false ;
+    # set to a logfile path to enable logging into log file
+    # skosmos:logFileName "" ;
+    # a default location for Twig template rendering
+    skosmos:templateCache "/tmp/skosmos-template-cache" ;
+    # customize the css by adding your own stylesheet
+    skosmos:customCss "resource/css/stylesheet.css" ;
+    # default email address where to send the feedback
+    skosmos:feedbackAddress "" ;
+    # email address to set as the sender for feedback messages
+    skosmos:feedbackSender "" ;
+    # email address to set as the envelope sender for feedback messages
+    skosmos:feedbackEnvelopeSender "" ;
+    # whether to display the ui language selection as a dropdown (useful for cases where there are more than 3 languages) 
+    skosmos:uiLanguageDropdown false ;
+    # whether to enable the spam honey pot or not, enabled by default
+    skosmos:uiHoneypotEnabled true ;
+    # default time a user must wait before submitting a form
+    skosmos:uiHoneypotTime 5 ;
+    # plugins to activate for the whole installation (including all vocabularies)
+    skosmos:globalPlugins () .
+
+# Skosmos vocabularies
+
+:ysa a skosmos:Vocabulary, void:Dataset ;
```

Comment:
Could we use the configuration for STW and UNESCO here as well? I'm a bit nervous about referring to the `api.dev.finto.fi` endpoint here. Of course you then have to give them some SPARQL endpoint URL that you assume exists...maybe a Fuseki running on the host system in port 3030?

In the past we've had situations where someone uses an old Skosmos version which has YSA/YSO in the default configuration, and that bombards our endpoint with heavy queries that used to work fine on some older Fuseki version but not with a newer version. (And believe me, this kind of thing where a particular query that used to be fast suddenly becomes very heavy after a Fuseki update has happened quite a few times!)

We should probably change the default `config.ttl` that is included with Skosmos as well - that's a separate task from this, but at least we shouldn't have more copies of a configuration file with YSA and YSO from the `api.dev.finto.fi` endpoint.

Reply:
Sorry, I was reviewing the comments from the bottom up, and read this after I commented in the one below.

Let me see if I can find a way to map an external `<some-host-or-localhost>:3030` to a container created with this `Dockerfile.ubuntu` and `config-docker.ttl`.

If we manage to do that, we might be able to have a single `config-docker.ttl` used by both Docker command, and `docker-compose` command.